api/conf/conf.go

```diff
@@ -149,6 +156,17 @@ func setConf() {
 			}
 		}
 
+		// access log
+		if config.Conf.Log.AccessLog.FilePath != "" {
+			AccessLogPath = config.Conf.Log.AccessLog.FilePath
+		}
+		if !filepath.IsAbs(AccessLogPath) {
+			AccessLogPath, err = filepath.Abs(WorkDir + "/" + AccessLogPath)
```

Comment:
Use `filepath.Join` instead of concatenating them manually.

Reply:
fixed.